flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -207,9 +209,8 @@ public void testCleanUpExceptionSuppressing() throws Exception {
 			testHarness.waitForTaskCompletion();
 		}
 		catch (Exception ex) {
-			if (!ExceptionUtils.findThrowable(ex, ExpectedTestException.class).isPresent()) {
-				throw ex;
-			}
+			assertTrue(ex.getCause() instanceof ExpectedTestException);
```

Comment:
nit: Unless there is some good reason 
`(!ExceptionUtils.findThrowable(ex, ExpectedTestException.class).isPresent())` is better to have compared to `ex.getCause() instanceof ExpectedTestException`. This good reason might be that you want to ensure and make it part of the contract, that the original exception is not wrapped - if that's the intention, feel free to ignore this comment :)

Reply:
> nit: Unless there is some good reason
> 
> ```
> (!ExceptionUtils.findThrowable(ex, ExpectedTestException.class).isPresent())
> ```
> 
> is better to have compared to
> 
> ```
> ex.getCause() instanceof ExpectedTestException
> ```
> 
> This good reason might be that you want to ensure and make it part of the contract, that the original exception is not wrapped - if that's the intention, feel free to ignore this comment :)

Yeah, I want to make sure the original exception is not wrapped, and disposed exception is the (only) suppressed one.